src/server/graphql/models/User/userMutation.js

```diff
@@ -151,7 +151,13 @@ export default {
         createdAt: new Date(userInfo.created_at),
         tms: userInfo.tms
       };
-      const {email, id: userId, picture, preferredName} = auth0User;
+      const {email, id: userId, createdAt, picture, preferredName} = auth0User;
```

Comment:
I stuck it before the conditional because we're either going to next update the user or insert the user. Can you give me rationale for sticking it lower? I'm not following...